src/canvas/SelectableCanvas.ts

```diff
@@ -729,6 +730,16 @@ export class SelectableCanvas<EventSpec extends CanvasEvents = CanvasEvents>
         // check pointer is over active selection and possibly perform `subTargetCheck`
         this.searchPossibleTargets([activeObject], pointer)
       ) {
+        // If you hold down selection key, you can select elements that are blocked by active selection.
+        if (this._isSelectionKeyPressed(e)) {
+          const other = this.searchPossibleTargets(this._objects, pointer);
+          if (
+            other &&
+            !(activeObject as ActiveSelection).contains(other, true)
+          ) {
+            return other;
+          }
+        }
```

Comment:
Does this works 100% of the time? this._objects includes the object in the active selection i assume.

Reply:
```ts
1.    else if (
        aObjects.length > 1 &&
        // check pointer is over active selection and possibly perform `subTargetCheck`
        this.searchPossibleTargets([activeObject], pointer)
      ) {
        // If you hold down selection key, you can select elements that are blocked by active selection.
        if (this._isSelectionKeyPressed(e)) {
          const other = this.searchPossibleTargets(this._objects, pointer);
          if (
            other &&
 2.           !(activeObject as ActiveSelection).contains(other, true)
          ) {
            return other;
          }
        }
        // active selection does not select sub targets like normal groups
3.        return activeObject;
      } 
```
This is the full text.
At point 1, it can be confirmed that it is definitely the activeSelection.
At point 2, if we assume that this._objects includes the activeSelection and other is the activeSelection, the result returned will be the same as at point 3.

Reply:
I'll try to reproduce on canvas the case i think may fail